### DISCUSSION TITLE
Can we expect MDS3 support at some point?

### DISCUSSION THREAD

**Discussion by daniel-mezo** [webauthn4j/webauthn4j#585] [Q&A] [answered]
Hi devs, @ynojima,

MDS3 has been introduced to FIDO2 for some time now, and it does not really seem to be compatible with MDS2. Furthermore MDS2 will be shut down in October 2022.
See this blog post: https://medium.com/webauthnworks/webauthn-fido2-whats-new-in-mds3-migrating-from-mds2-to-mds3-a271d82cb774

As we are integrating webauthn4j at the moment, it would be interesting for us what is the timeline (if there is any) to add MDS3 support for the library.

Thanks in advance for the replies,
Daniel

**Comment by ynojima** [comment, marked as answer]
I'd like to implement it (Tracking issue: https://github.com/webauthn4j/webauthn4j/issues/564 ). 
I don't have concrete timeline, but I'll complete it before the shutdown of MDS2.
